.circleci/config.yml

```diff
@@ -155,7 +155,7 @@ jobs:
           <<: *defaults_license_scanner
       - checkout
       - restore_cache:
-          key: dependency-cache-5-{{ checksum "src/package.json" }}
+          key: dependency-cache-{{ checksum "src/package.json" }}
```

Comment:
Can you make this rely on `package-lock.json`?

Reply:
fixed